openapi_python_client/parser/properties/__init__.py

```diff
@@ -256,6 +256,7 @@ def _string_based_property(
             default=convert("datetime.datetime", data.default),
             nullable=data.nullable,
             description=data.description,
+            read_only=data.readOnly,
```

Comment:
I had to add this attribute to all of the `__Property` classes because otherwise the Jinja templates have no way to access it. It would be nice if these objects just preserved a reference to the original schema data, but they don't (at least in this version).